src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -9,6 +9,7 @@ Release History
 * `az aks get-credentials --admin` won't replace the user context in $HOME/.kube/config
 * expose read-only "nodeResourceGroup" property on managed clusters
 * fix `az acs browse` command error
+* Minor fixes
```

Comment:
This boilerplate is only needed if there are no other entries for that version. Since there are, you don't need anything in HISTORY.RST.